crates/red_knot_python_semantic/src/types/infer.rs

```diff
@@ -5452,7 +5486,7 @@ mod tests {
         // TODO: once we support `sys.version_info` branches,
         // we can set `--target-version=py311` in this test
         // and the inferred type will just be `BaseExceptionGroup` --Alex
```

Comment:
We can now remove this comment, and the similar one in two below tests, because the lack of `sys.version_info` handling no longer results in a spurious `Unknown` here.